packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -243,6 +243,12 @@ location ~ ^/service/(?<service_path>.+) {
     set $service_tag '';
     set $original_uri $uri;
 
+    # Give services a lot of leeway. A particular extreme is for uploading
+    # packages to a package registry. The Jupyter package is one of our
+    # largest at just under 10GB. Use 16GB as the limit to allow for this to
+    # expand.
+    client_max_body_size 16384m;
```

Comment:
Thanks for the code comment!

Reply:
I like to know why a piece of configuration is set how it is.